bladex/utils/params.py

```diff
@@ -0,0 +1,327 @@
+
+        params = ['chord', 'pitch', 'rake', 'skew', 'camber']
+        for section, param in zip(section_all, params):
+            nbasis_all[param] = config.getint(section, 'number of control points (spline knots)')
+            degree_all[param] = config.getint(section, 'spline degree')
+
+            coordinates_all = config.get(section, 'control points coordinates')        
+
+            # Non empty strings return boolean True
+            if bool(coordinates_all) is True:
+                lines = coordinates_all.split('\n')
+                control_points_all[param] = np.zeros((len(lines), 2))
+                for line, j in zip(lines, list(range(0, nbasis_all[param]))):
+                    values = line.split()
+                    control_points_all[param][j] = np.array(
+                                                [float(values[0]),
+                                                float(values[1])])
+            deformations = config.get(section, 'control points Y-deformations')
+            if bool(deformations) is False:
+                raise ValueError('control points Y-deformations in section [' + section + '] must be non-empty.')
+            lines = deformations.split('\n')
+            deformations_all[param] = np.zeros(len(lines))
+            for line, j in zip(lines, list(range(0, nbasis_all[param]))):
+                deformations_all[param][j] = np.array([float(line)])
+
+
+        # Back copy from the dictionaries into the class members
+        self.nbasis_chord = nbasis_all['chord']
+        self.degree_chord = degree_all['chord']
+        self.control_points_chord = control_points_all['chord']
+        self.deformations_chord = deformations_all['chord']
+
+        self.nbasis_pitch = nbasis_all['pitch']
+        self.degree_pitch = degree_all['pitch']
+        self.control_points_pitch = control_points_all['pitch']
+        self.deformations_pitch = deformations_all['pitch']
+
+        self.nbasis_rake = nbasis_all['rake']
+        self.degree_rake = degree_all['rake']
+        self.control_points_rake = control_points_all['rake']
+        self.deformations_rake = deformations_all['rake']
+
+        self.nbasis_skew = nbasis_all['skew']
+        self.degree_skew = degree_all['skew']
+        self.control_points_skew = control_points_all['skew']
+        self.deformations_skew = deformations_all['skew']
+
+        self.nbasis_camber = nbasis_all['camber']
+        self.degree_camber = degree_all['camber']
+        self.control_points_camber = control_points_all['camber']
+        self.deformations_camber = deformations_all['camber']
+
+    def write_parameters(self, filename='parameters.prm'):
+        """
+        This method writes a parameters file (.prm) called `filename` and fills
+        it with all the parameters class members. Default value is
+        parameters.prm.
+        :param string filename: parameters file to be written out.
```

Comment:
I think that here you need a return in order to proper generate the doc. Same in other methods.